features/buyer/buyer_create_briefs.feature

```diff
@@ -1,59 +1,186 @@
-@not-production @functional-test @wip
-Feature: Create new buyer account
+@not-production @functional-test
+Feature: Buyer create briefs
 
-#Scenario: User steps through supplier account creation process
-#  Given I am on the 'Digital Marketplace' landing page
-#  When I click 'Create supplier account'
-#  Then I am on the 'Create supplier account' page
+Background: Login to Digital Marketplace as the newly created buyer
+  Given I have a buyer user account
+  When I have logged in to Digital Marketplace as a 'Buyer' user
 
-Scenario: Publish a brief for an individual specialist
+Scenario: Start brief creation for an individual specialist
   Given I am on the 'Digital Marketplace' landing page
-  When I click 'Find an individual specialist'
-  Then I am taken to the 'appropriate page'
+  When I click the 'Find an individual specialist' link
+  Then I am taken to the buyers 'Find an individual specialist' page
 
-  When I choose the 'appripriate' role
-  And I click 'Save and continue'
-  Then I am taken to the 'appropriate page'
+  When I click the 'Choose specialist role' button
+  Then I am taken to the 'Requirements title' page
+
+  When I enter 'Find an individual specialist brief' in the 'title' field
+  And I click the 'Save and continue' button
+  Then I am taken to the 'Location' page
 
-  When I choose the 'appropriate' location
+  When I choose 'North East England' for 'location'
   And I click 'Save and continue'
-  Then I am taken to the 'appropriate page'
+  Then I should be on the "Overview of work" page for the buyer brief 'Find an individual specialist brief'
+  @wip
+Scenario: Newly created brief should be listed on the buyer's dashboard and the count of unanswered questions is correct
+  Given I am logged in as 'DM Functional Test Buyer User 1' 'Buyer' user and am on the dashboard page
+  Then The buyer brief 'Find an individual specialist brief' is '' listed on the buyer's dashboard
```

Comment:
Having empty parameters in the feature descriptions seems a bit odd to me. I don't think it's been done before?  Might be nice to avoid introducing it by rewording the definition somehow.